### DISCUSSION TITLE
Pull request doesnt work using Tortoise GIT after using PAT as password

### DISCUSSION THREAD

**Discussion by melwindevassy** [community/community#24713] [Apps, API and Webhooks]
<p>I generated a PAT token in GIT HUB and when i try to use it in Tortoise GIT Pull it keeps erroring out can someone please help</p>

**Comment by tajmone** [comment]
<p>You need to delete your previously stored credentials, so you can start using your PAT instead of the old password. Where and how these credentials are stored might depend on many factors: your OS, which Git package you’re using, and the various Git apps involved.</p>

**Comment by melwindevassy** [comment, marked as answer]
<p>Hi All, i created a new PAT token and used it as the password and it started working.<br>
Thank you for your help</p>
